spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/SkipperController.java

```diff
@@ -61,6 +62,9 @@
 	@Value("${info.app.version:#{null}}")
 	private String appVersion;
 
+	@Autowired
+	private SkipperStateMachineService stateMachineService;
```

Comment:
Shouldn't have any autowiring and we create the controller in the SkipperServerConfiguration class.  Will modify on merge.

Reply:
It feels a bit awkward now that the Controller calls both `SkipperStateMachineService` and `ReleaseService` since  `SkipperStateMachineService` also calls `ReleaseService`.   Taking the `ReleaseService` methods ` list, history, manifest, status` and putting them in a new interface would make these relationships more direct in that `ReleaseService` only exposes the methods that are "orchestrated" by `SkipperStateMachineService`